vncviewer/Viewport.cxx

```diff
@@ -243,6 +243,44 @@ static const char * dotcursor_xpm[] = {
   " ... ",
   "     "};
 
+static const char * largecursor_xpm[] = {
```

Comment:
If we're going to use a standard pointer, then I think we get a better result if we just use the standard system pointer.

Reply:
I'm not entirely sure what you mean by "standard system pointer" - I was trying to follow the way this is done just a few lines earlier for the dot cursor, but if there's a better way of doing this, I'm open to using it.

Reply:
For FLTK that means calling `cursor(FL_CURSOR_DEFAULT);`.